backend/src/main/java/com/example/backend/service/WorkService.java

```diff
@@ -15,4 +17,22 @@ public class WorkService {
     public List<WorkDto> getAllWorksByCarId (String carId) {
         return workRepository.findAllByCarId(carId);
     }
+
+    public WorkDto createWork(WorkDto workDto) {
+        Work work = new Work(workDto.id(), workDto.carId(), workDto.type(), workDto.mileage(), workDto.date(), workDto.price());
+        workRepository.save(work);
+        return workDto;
+    }
+
+    public WorkDto updateWork(String id, WorkDto workDto) {
+        Work work = workRepository.findById(id).orElseThrow(NoSuchElementException::new);
+        Work newWork = new Work(work.id(), workDto.carId(), workDto.type(), workDto.mileage(), workDto.date(), workDto.price());
+        workRepository.save(newWork);
+        return workDto;
```

Comment:
The input parameter is returned without any changes.

Reply:
Agree with that, will fix it